idd/Energy+.idd.in

```diff
@@ -38873,34 +38870,22 @@ AirTerminal:SingleDuct:VAV:Reheat:VariableSpeedFan,
        \minimum 0.0
        \maximum 1.0
        \note fraction of cooling air flow rate
-  A3,  \field Air Inlet Node Name
```

Comment:
I implemented option 2.  The air inlet and outlet nodes are mined as is.  But I added a code that compares the air inlet and out let node names mined with those specified by user.  If there is mismatch warning error message will be thrown displaying user specified and the expected node names. The simulation continues.  Below is a sample warning message:
   ** Warning ** GetSysInput: invlid air terminal object air inlet node name in AirTerminal:SingleDuct:VAV:Reheat:VariableSpeedFan = SPACE1-1 VAV REHEAT
   **   ~~~   **  Specified air inlet node name is = SPACE1-1 ATU IN NODE_BN.
   **   ~~~   **  Expected air inlet node name is = SPACE1-1 ATU IN NODE.